src/Team/Team.js

```diff
@@ -0,0 +1,43 @@
+import { $teamListContainer } from '../etc/dom.js';
+import { loadTeamList } from './loadTeam.js';
+import { createTeam } from './createTeam.js';
+import { removeTeam } from './removeTeam.js';
+
+export const team = async () => {
+  await loadTeamList();
+
+  $teamListContainer.addEventListener('click', filter);
+};
+
+team();
+
+const filter = ({ target }) => {
+  const targetClass = target.classList.item(0);
+
+  if (targetClass === 'destroy') {
+    removeTeam(target);
+  } else if (targetClass === 'ripple') {
+    createTeam();
+  } else {
```

Comment:
```suggestion
  if (target.classList.contains('destroy')) {
    removeTeam(target);
  } else if (target.classList.contains('ripple')) {
    createTeam();
  } else {
```

이 부분은 `classList.contains()`메서드를 사용해서 위와 같이 수정하면 더 안전하고 깔끔한 코드가 될거 같습니다!

Reply:
contains를 쓰면 더 안전한 이유가 뭔지 여쭤봐도 될까요?

Reply:
classList에 index로 접근해서 class를 가져오게 되면 해당 요소 앞에 새로운 class가 추가 되었을 때 원하는 class를 `targetClass`에 담지 못할 수도 있습니다.

하지만 `classList.contains()` 메서드를 사용하게 되면 해당 클래스가 전체 클래스리스트의 몇번째에 위치하는지와 관계없이 해당 클래스를 포함하는지 확인할 수 있어 이후 새로운 클래스가 추가되는 경우에도 더 안전하게 확인할 수 있습니다!